.github/workflows/linter.yml

```diff
@@ -28,3 +28,13 @@ jobs:
           VALIDATE_YAML: false
```

Comment:
Actually one comment, are we keeping superlinter + ruff? Maybe remove super linter? Or is it so broken right now that it's not even running?

Reply:
We were thinking that it should be kept until the presrat merge (and gui would figure out the pylint issue) so we didnt break too many things. Some inline overrides will need to be added to appease ruff when we fully adopt

Reply:
Good point. We don't have a match 1:1 between both tools. I was planning on keeping superlinter for a short time to see what they disagree so we can create an opinion on what is missing and should be included or dropped.

I don't know what is the issue with superlinter right now, but it don't hurt to keep it for a short time. I'll move to PresRat after this, and can return to superlinter after that.

Reply:
It running now, and the only complain is with a yaml file and I couldn't understand what is the issue.

Reply:
Fixed! Closing this one.